plugins/splunk-on-call/schema.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  /**
+   * Splunk On Call Plugin specific configs
+   */
+  splunkOnCall: {
```

Comment:
Think this should be `splunkOnCall?` instead to make it optional and it will fix the tests.

https://github.com/backstage/backstage/blob/ae903f8e718d89cdff5538bf2d5c93ec7c59c360/plugins/splunk-on-call/src/api/client.ts#L46 doesn't require it anyway it uses `optionalString`

That will also fix the tests :)